packages/suite/src/components/wallet/WalletLayout/WalletLayout.tsx

```diff
@@ -32,7 +23,8 @@ const EmptyHeaderPlaceholder = styled.div`
 type WalletLayoutProps = {
     title: ExtendedMessageDescriptor['id'];
     account: AppState['wallet']['selectedAccount'];
-    showEmptyHeaderPlaceholder?: boolean;
+    isSubpage?: boolean;
+    showEmptyHeaderPlaceholder?: boolean; // TODO: remove this, it's not even used according to it's name (account details)
```

Comment:
If we really can remove it, then lets remove it.

Reply:
@jvaclavik I meant that this spacing should be integrated into the layout, because using a prop for rendering some space just doesn't sound sustainable or that obvious to me :(